docs/install/README.md

```diff
@@ -38,6 +38,10 @@ component.
 
 [Install with Gloo](./Knative-with-Gloo.md): Gloo functions as a lightweight gateway for Knative. Choose this option if you don't require a service mesh in your cluster and want a lightweight alternative to Istio. Gloo supports all documented Knative features, as well as extensions to Serving such as Eventing and Monitoring.
 
+## Installing Knative with Kourier
```

Comment:
We're doing the headings in alphabetical order; could you put Kourier after the "Installing Knative with Istio" heading?